src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkUnrecognizedSectionInspection.kt

```diff
@@ -59,7 +59,9 @@ class SmkUnrecognizedSectionInspection : SnakemakeInspection() {
     }
 
     override fun createOptionsPanel(): JComponent? {
-        return ListEditForm(SnakemakeBundle.message("INSP.NAME.section.unrecognized.ignored"),
-            ignoredItems).contentPanel
+        return ListEditForm(
```

Comment:
Please reformat according to Kotlin code style conventions:
> Prefer using an expression body for functions with the body consisting of a single expression.

See https://kotlinlang.org/docs/coding-conventions.html#functions